change/@ni-nimble-components-8dfe3583-abfb-4385-a130-bf75e17f7ed5.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Didn't see it in the PR description so just to make sure, will there be a follow-up with Angular and Blazor changes?

Reply:
Could you update the PR description to mention #632 (and mark it as fixed if you add Angular and Blazor to this PR)?

Reply:
Yes, I'll do those in a follow-on PR. Updated the PR description.